Source/FieldSolver/FiniteDifferenceSolver/EvolveEPML.cpp

```diff
@@ -151,10 +190,14 @@ void FiniteDifferenceSolver::EvolveEPMLCartesian (
 
                 Ey(i, j, k, PMLComp::yx) -= c2 * dt * (
                     T_Algo::DownwardDx(Bz, coefs_x, n_coefs_x, i, j, k, PMLComp::zx)
-                  + T_Algo::DownwardDx(Bz, coefs_x, n_coefs_x, i, j, k, PMLComp::zy) );
+                  + T_Algo::DownwardDx(Bz, coefs_x, n_coefs_x, i, j, k, PMLComp::zy)
+                  - sin_th*cos_th*(T_Algo::DownwardDy(By, coefs_y, n_coefs_y, i, j, k, PMLComp::yz)
+                  + T_Algo::DownwardDy(By, coefs_y, n_coefs_y, i, j, k, PMLComp::yx)) );
```

Comment:
One comment discussed offline: for each of the new terms (like dBy/dy here), one needs to check that the correct finite difference function ("downward" v. "upward") is used, depending on whether the field, which the derivative is applied to, is nodal or cell-centered along the direction of that derivative.